### PR TITLE
[OCTRL-161] Add a description field to the main workflow templates

### PR DESCRIPTION
**Pull request by claireguyot**

### CODE REVIEW

**Comment by vascobarroso** [comment]
@knopers8 could you provide a more detailed description of the 2 QC workflows for @claireguyot to add ?

File: workflows/qc-postprocessing.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name: !public qc-postprocessing
+description: !public "Qc-postprocessing worflow template"
```
Comment (knopers8) [inline workflows/qc-postprocessing.yaml:2]: A workflow template for running QC post-processing as OCC State Machine (obsolete)

File: workflows/qc-server-workflow.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 name: !public qc-server-workflow
+description: !public "Qc-server-workflow worflow template"
```
Comment (knopers8) [inline workflows/qc-server-workflow.yaml:2]: A workflow template for running standalone QC workflows on QC servers.

File: workflows/qc-postprocessing.yaml (hunk 1 above)
Comment (vascobarroso) [reply workflows/qc-postprocessing.yaml:2]: @knopers8 by obsolete does it mean it we can remove it ?

Comment (knopers8) [reply workflows/qc-postprocessing.yaml:2]: It seems that `qc-server-workflow` does not cause problems, so we can indeed remove it. I will make a PR.

File: workflows/qc-server-workflow.yaml (hunk 2 above)
Comment (vascobarroso) [reply workflows/qc-server-workflow.yaml:2]: @claireguyot could you update this description with the suggestion from @knopers8 ?

File: workflows/qc-postprocessing.yaml (hunk 1 above)
Comment (vascobarroso) [reply workflows/qc-postprocessing.yaml:2]: Ok thanks. @claireguyot I guess you will need to rebase and drop these changes.

File: workflows/qc-server-workflow.yaml (hunk 2 above)
Comment (claireguyot) [reply workflows/qc-server-workflow.yaml]: @vascobarroso The changes have been added!

**Review by vascobarroso — APPROVED** [review]

**Pull request merged**